### DISCUSSION TITLE
Windows resizing

### DISCUSSION THREAD

**Discussion by KainSkybrace** [valinet/ExplorerPatcher#262] [Ideas]
First, thank you so much for creating this amazing and yet super easy to use tool!
To get back to the functionnalities I was using in Win 10, I cannot resize automatically the windows now, as I could do in Win 10 (with the mouse). Looks like there is a new way to do so, but it does not work, and I am assuming it is because of the dll.
Is it something you intend to fix?
That would be amazing :-)
Thank you

**Comment by valinet** [comment]
What do you mean resizing automatically? Also, the answer to whether the DLL interferes with anything is to take it out, test on a clean slate, then reinstall and see if you still have problem. Easy to test and then you can report back whether it indeed is a problem or not.

Thanks
